### PR TITLE
#1795 set valid from and until

### LINKED ISSUE #1795: Add setValidFromAndUtil update action for CartDiscount & DiscountCode

### PR DESCRIPTION
**Pull request by katmatt**
# Description

Closes #1795 

# Checklist

- [x] Update release Notes
- [x] Add to the current github milestone

### CODE REVIEW

**Comment by katmatt** [comment]
@lauraluiz The failing test isn't related to my change and fails because of an issie with our azure subscription.

File: commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java [hunk 1]
```diff
@@ -147,9 +147,11 @@
...
     <li class=change-in-release>{@link io.sphere.sdk.producttypes.commands.updateactions.AddAttributeDefinition#of(AttributeDefinition)} changed parameter to {@link io.sphere.sdk.producttypes.commands.updateactions.AddAttributeDefinition#of(AttributeDefinitionDraft)} instead</li>
-    <li class=new-in-release>now possible to change customer in order via {@link io.sphere.sdk.orders.commands.updateactions.SetCustomerId} which would result in a {@link io.sphere.sdk.orders.messages.OrderCustomerSetMessage}</li>
+    <li class=new-in-release>It's now possible to change customer in order via {@link io.sphere.sdk.orders.commands.updateactions.SetCustomerId} which would result in a {@link io.sphere.sdk.orders.messages.OrderCustomerSetMessage}</li>
+    <li  class=new-in-release>New update action {@link io.sphere.sdk.cartdiscounts.commands.updateactions.SetValidFromAndUntil} to {@link ProductDiscount}</li>
+    <li  class=new-in-release>New update action {@link io.sphere.sdk.discountcodes.commands.updateactions.SetValidFromAndUntil} to {@link ProductDiscount}</li>
```
Comment (lauraluiz) [inline commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java]: Both notes say the same, was it maybe CartDiscount?

File: commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java [hunk 2]
```diff
@@ -147,9 +147,11 @@
...
-    <li class=new-in-release>new update action {@link io.sphere.sdk.productdiscounts.commands.updateactions.SetValidFromAndUntil} to {@link ProductDiscount}</li>
+    <li class=new-in-release>New update action {@link io.sphere.sdk.productdiscounts.commands.updateactions.SetValidFromAndUntil} to {@link ProductDiscount}</li>
     <li class=change-in-release>{@link io.sphere.sdk.producttypes.commands.updateactions.AddAttributeDefinition#of(AttributeDefinition)} changed parameter to {@link io.sphere.sdk.producttypes.commands.updateactions.AddAttributeDefinition#of(AttributeDefinitionDraft)} instead</li>
-    <li class=new-in-release>now possible to change customer in order via {@link io.sphere.sdk.orders.commands.updateactions.SetCustomerId} which would result in a {@link io.sphere.sdk.orders.messages.OrderCustomerSetMessage}</li>
+    <li class=new-in-release>It's now possible to change customer in order via {@link io.sphere.sdk.orders.commands.updateactions.SetCustomerId} which would result in a {@link io.sphere.sdk.orders.messages.OrderCustomerSetMessage}</li>
```
Comment (lauraluiz) [inline commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java]: +1 for the fixes :D

File: commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java (hunk 1 above)
Comment (katmatt) [reply commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java]: Good catch 👍 Fixed: should be `CartDiscount` and `DiscountCode`.

Comment (katmatt) [reply commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java]: Good catch 👍 Fixed: should be `CartDiscount` and `DiscountCode`.

**Pull request merged**